src/EnergyPlus/SingleDuct.cc

```diff
@@ -2099,10 +2099,10 @@ namespace SingleDuct {
 		}
 		if ( Sys( SysNum ).ZoneMinAirFracMethod == ConstantMinFrac ) {
 			if ( ZoneSizingRunDone ) {
-				if ( CurZoneEqNum > 0 ) {
+				if ( CurTermUnitSizingNum > 0 ) {
```

Comment:
Based on the issue description, I guess this gets the index correctly due to the extra TermUnitSizing objects being stored.